packages/core/src/types/component.ts

```diff
@@ -1,8 +1,8 @@
-import { Observable } from 'rxjs';
+import { Observable, OperatorFunction } from 'rxjs';
 import { PipcookPlugin } from './plugins';
```

Comment:
I have modified this file, could you please pull and merge it?

Reply:
Sure!